.changeset/nervous-plums-travel.md

```diff
@@ -0,0 +1,8 @@
+---
+'@xstate/solid': major
+---
+
+- Add solidjs xstate adaptors with useMachine, useActor, useSelector, useSpawn, useInterpret, and FSM useMachine/useService
+- Add tests for xstate-solid
+- Add documentation for xstate-solid package
+- Add xstate-solid templates to examples
```

Comment:
Those mostly mention changes/work done in this PR - this message ends up being in the CHANGELOG though and thus it only should contain what is relevant for readers of the CHANGELOG.

In fact, for the very first release of this package we can probably just skip adding a changeset altogether